src/components/non-bitgo.js

```diff
@@ -139,16 +143,19 @@ class NonBitGoRecoveryForm extends Component {
       };
 
       // Retrieve the desired file path and file name
-      const filePath = dialog.showSaveDialog(dialogParams);
+      const filePath = await dialog.showSaveDialog(dialogParams);
       if (!filePath) {
         // TODO: The user exited the file creation process. What do we do?
         return;
       }
 
-      fs.writeFileSync(filePath, JSON.stringify(recovery, null, 4), 'utf8');
+      fs.writeFileSync(filePath.filePath, JSON.stringify(recovery, null, 4), 'utf8');
 
-      this.setState({ recovering: false, done: true, finalFilename: filePath });
+      this.setState({ recovering: false, done: true, finalFilename: [filePath.filePath] });
+      alert('We recommend that you use a third-party API to decode your txHex a' +
+            'nd verify its accuracy before broadcasting.');
```

Comment:
can we just make the 'and' a whole word on the nextline?

Reply:
we needed to break the sentence because it is over 80 char long

Reply:
We can use something like 

```
[
`We recommend that you use a third-party API to decode`
`your txHex and verify its accuracy before broadcasting`
].join(''
```